lib/uploadcare/resources/file.rb

```diff
@@ -95,7 +95,7 @@ def is_deleted?
       # copy file to target location
       # note what file copied with operations
       def copy with_operations=true, target=nil
-        warn "[DEPRECATION] `copy` is deprecated.  Please use `internal_copy` or `external_copy` instead."
+        warn "[DEPRECATION] `copy` is deprecated and will be removed in version 3.0.  Please use `internal_copy` or `external_copy` instead."
```

Comment:
@dmitry-mukhin come on, we are not going to maintain this piece of code anymore. Why should we care about it's readability?